src/ui/server/controllers/insurance/your-buyer/validation/rules/first-name.test.ts

```diff
@@ -0,0 +1,48 @@
+import firstNameRule from './first-name';
+import FIELD_IDS from '../../../../../constants/field-ids/insurance/your-buyer';
+import { ERROR_MESSAGES } from '../../../../../content-strings';
+import emptyFieldValidation from '../../../../../shared-validation/empty-field';
+import { RequestBody } from '../../../../../../types';
+
+const {
+  COMPANY_OR_ORGANISATION: { FIRST_NAME: FIELD_ID },
+} = FIELD_IDS;
+
+const {
+  INSURANCE: {
+    YOUR_BUYER: {
+      COMPANY_OR_ORGANISATION: { [FIELD_ID]: ERROR_MESSAGE },
+    },
+  },
+} = ERROR_MESSAGES;
+
+describe('controllers/insurance/your-buyer/validation/first-name', () => {
+  const mockErrors = {
+    summary: [],
+    errorList: {},
+  };
+
+  const mockBody = {
+    [FIELD_ID]: '',
+  } as RequestBody;
+
+  describe(`when ${FIELD_ID} field is not provided`, () => {
+    it('should return `emptyFieldValidation`', () => {
+      const result = firstNameRule(mockBody, mockErrors);
+
+      const expected = emptyFieldValidation(mockBody, FIELD_ID, ERROR_MESSAGE.IS_EMPTY, mockErrors);
+
+      expect(result).toEqual(expected);
+    });
+  });
+
+  describe(`when ${FIELD_ID} field is provided`, () => {
+    it('should not return validation errors', () => {
+      mockBody[FIELD_ID] = 'test';
+
+      const result = firstNameRule(mockBody, mockErrors);
+
+      expect(result).toEqual(mockErrors);
+    });
+  });
```

Comment:
This test can just have one `it`, this helper function handles both scenarios 😄  Same for `last-name`

```js
it('should return the result of emptyFieldValidation', () => {
```